lib/opensusebasetest.pm

```diff
@@ -318,6 +318,86 @@ sub handle_uefi_boot_disk_workaround {
     wait_screen_change { send_key 'ret' };
 }
 
+=head2 wait_grub
+
+  wait_grub([bootloader_time => $bootloader_time] [,in_grub => $in_grub]);
+
+Makes sure the bootloader appears. Returns successfully when reached the bootloader menu, ready to control it further or continue. The time waiting for the bootloader can be configured with
+C<$bootloader_time> in seconds. Set C<$in_grub> to 1 when the
+SUT is already expected to be within the grub menu.
+=cut
+sub wait_grub {
+    my ($self, %args) = @_;
+    my $bootloader_time = $args{bootloader_time} // 100;
+    my $in_grub         = $args{in_grub}         // 0;
+    my @tags            = ('grub2');
+    push @tags, 'bootloader-shim-import-prompt'   if get_var('UEFI');
+    push @tags, 'boot-live-' . get_var('DESKTOP') if get_var('LIVETEST');             # LIVETEST won't to do installation and no grub2 menu show up
+    push @tags, 'bootloader'                      if get_var('OFW');
+    push @tags, 'encrypted-disk-password-prompt'  if get_var('ENCRYPT');
+    push @tags, 'linux-login'                     if get_var('KEEP_GRUB_TIMEOUT');    # Also wait for linux-login if grub timeout was not disabled
+    if (get_var('ONLINE_MIGRATION')) {
+        push @tags, 'migration-source-system-grub2';
+    }
+    # after gh#os-autoinst/os-autoinst#641 68c815a "use bootindex for boot
+    # order on UEFI" the USB install medium is priority and will always be
+    # booted so we have to handle that
+    # because of broken firmware, bootindex doesn't work on aarch64 bsc#1022064
+    push @tags, 'inst-bootmenu'
+      if (get_var('USBBOOT') && get_var('UEFI')
+        || (check_var('ARCH', 'aarch64') && get_var('UEFI'))
+        || get_var('OFW')
+        || (check_var('BOOTFROM', 'd')));
+    $self->handle_uefi_boot_disk_workaround
```

Comment:
I hardly can read this , is there 'if' inside function call ? I know that it is not required by perl but can you please add () to function calls ?

Reply:
1. This is unrelated to this PR just moving this code. Please see the whole change and the corresponding removed parts
2. I agree that it's not easily readable. Please keep that in mind for review when other people mess around on this lines again :)